tests/test_tket_symbolic.py

```diff
@@ -138,12 +138,28 @@ def test_symbolic_numeric_blend() -> None:
     _test_circuit(circuit, symbols)
 
 
+def test_symbolic_numeric_blend_gate() -> None:
+    symbols = [Symbol("p0"), Symbol("p1"), Symbol("p2")]  # type: ignore
+
+    circuit = Circuit(1)
+    circuit.add_gate(OpType.U2, [symbols[0] * 3.0, symbols[1] + symbols[2] + 2], [0])
+
+    _test_circuit(circuit, symbols)
+
+    circuit = Circuit(1)
+    circuit.H(0)
+    circuit.Rx(symbols[2], 0)
+    circuit.add_gate(OpType.U2, [symbols[0], symbols[1]], [0])
+
+    _test_circuit(circuit, symbols)
+
+
 def test_not_in_qujaxgates() -> None:
     symbols = [Symbol("p0")]  # type: ignore
 
-    circuit = Circuit(2)
+    circuit = Circuit(3)
     circuit.Rx(symbols[0], 0)
-    circuit.ZZPhase(0.1, 0, 1)
+    circuit.XXPhase3(0.1, 0, 1, 2)
```

Comment:
Is there a reason why this is changed?

Reply:
Yes ZZPhase has been added to qujax in 0.2.8 and this test is supposed to test a gate not found in qujax